nautobot/docs/release-notes/version-1.0.md

```diff
@@ -4,15 +4,170 @@ This document describes all new features and changes in Nautobot 1.0, a divergen
+
+For more details, please refer to the GraphQL website, as well as to the [Nautobot GraphQL](../additional-features/graphql.md) documentation.
+
+#### Plugin API Enhancements
+
+Plugins can now provide custom [data validation](#data-validation-plugin-api) logic.
+
+Plugins can now include executable [Jobs](../additional-features/jobs.md) (formerly known as Custom Scripts and Reports) that will automatically be added to the list of available Jobs for a user to execute.
+
+Additional data models defined by a plugin are automatically made available in [GraphQL](../additional-features/graphql).
+
+Plugins can now define additional Django apps that they require and these dependencies will be automatically enabled when the plugin is activated.
+
+#### Single Sign-On / Social Authentication Support
+
+Nautobot now supports single sign on as an authentication option using OAuth2, OpenID, SAML, and others, using the [social-auth-app-django](https://python-social-auth.readthedocs.io/en/latest/) module. For more details please refer to the guide on [SSO authentication](../configuration/authentication/sso.md).
+
+#### User-Defined Relationships
+
+User-Defined, or "custom", [relationships](../../models/extras/relationship) allow users to create their own relationships between models in Nautobot to best suit the needs of their specific network design. Nautobot comes with opinionated data models and relationships.
+
+For example, a VLAN is mapped to a Site by default.  After a VLAN is created today, you then assign that VLAN to an Interface on a Device. This Device should be within the initial mapped Site.  However, many networks today have different requirements and relationships for VLANs (and many other models): VLANs may be limited to racks in Layer 3 DC fabrics; VLANs may be mapped to multiple buildings in a campus; they may span sites.  Other use cases include circuits, ASNs, or IP addressing--just to name a few--allowing users to define the exact relationships required for their network.
+
 ### Changed
 
+#### Code Reorganization
+
+All of the individual Django apps in NetBox (`dcim`, `extras`, `ipam`, etc.) have been moved into a common `nautobot` Python package namespace. The `netbox` application namespace has been moved to `nautobot.core`. This will require updates when porting NetBox custom scripts and reports to Nautobot jobs, as well as when porting NetBox plugins to Nautobot.
+
+#### Packaging Changes
+
+Nautobot is now packaged using [Poetry](https://python-poetry.org/) and builds as an installable Python package. `setup.py` and `requirements.txt` have been replaced with `pyproject.toml`. Releases of Nautobot are now published to [PyPI](https://pypi.org/), the Python Package Index, and therefore can now be installed using `pip install nautobot`.
```

Comment:
This is correct and accurate as a `Changed` entry, but I think it's a big enough thing that it's worth adding a note under the `Added` section *also* would be good, something like:

```
#### Installable Python Package

Nautobot is now installable as a self-contained Python package via `pip install nautobot`.
```